examples/l5pc/opt_l5pc.py

```diff
@@ -76,10 +76,11 @@ def main():
         logger.debug('Doing compile')
         import commands
         commands.getstatusoutput('cd mechanisms/; nrnivmodl; cd ..')
-
-    # TODO store definition dicts in json
-    # TODO rename 'score' into 'objective'
-    # TODO add functionality to read settings of every object from config format
+        #NEURON expects the compiled mechanisms to be in ./x86_64
```

Comment:
Actually, I don't think we need the 'cd' command.
We just need 'nrnivmodl mechanisms' in the root